[git-webkit] Support callbacks as main arguments

JonWBedard opened the pull request:
#### 4790532f26b0c40db73d2f04d7d2989ef89be2b3
```
[git-webkit] Support callbacks as main arguments
https://bugs.webkit.org/show_bug.cgi?id=229739
<rdar://problem/82597266>

Reviewed by NOBODY (OOPS!).

* Tools/Scripts/git-webkit:
(is_webkit_filter): Return item if repository is Webkit.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/__init__.py:
(main): Pass repository to contributors, identifier_template and subversion
to dynamically generate values if those arguments are callable.
```

rniwa commented:
We don't accept PR requests on Github as our main repository is still in Subversion.

This doesn't seem like something you meant to push to the main repository anyway?

JonWBedard committed:
[git-webkit] Support callbacks as main arguments
https://bugs.webkit.org/show_bug.cgi?id=229739
<rdar://problem/82597266>

Reviewed by NOBODY (OOPS!).

* Tools/Scripts/git-webkit:
(is_webkit_filter): Return item if repository is Webkit.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/__init__.py:
(main): Pass repository to contributors, identifier_template and subversion
to dynamically generate values if those arguments are callable.
